apps/backoffice-v2/src/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/utils/transformErrors.ts

```diff
@@ -0,0 +1,48 @@
+import { AnyObject } from '@ballerine/ui';
+import { RJSFValidationError } from '@rjsf/utils';
+
+export const transformErrors = (errors: RJSFValidationError[]): RJSFValidationError[] => {
```

Comment:
Do we transform errors differently in the backoffice compared to the KYB app or workflows service? Also, check out `Ajv.errorsText` as it might already provide a good format.